runtime/oti/VMHelpers.hpp

```diff
@@ -2058,12 +2058,12 @@ class VM_VMHelpers
 	}
 
 	static VMINLINE bool
-	needScanStacksForContinuation(J9VMThread *vmThread, j9object_t continuationObject) {
+	needScanStacksForContinuation(J9VMThread *vmThread, j9object_t continuationObject, bool bSTWScan = false) {
```

Comment:
I'd rename this parameter to scanOnly(Dis?)Mounted. Callers will know when it's important to take that into consideration  (and pass true/false accordingly). STW has no meaning in this context.